.github/workflows/iroha2-dev-pr.yml

```diff
@@ -90,6 +86,64 @@ jobs:
         run: bash -c './scripts/test_docker_compose.sh || ( docker-compose logs; false )'
       - name: Cleanup docker test environment
         run: ./scripts/cleanup_docker_test_env.sh
+  echo-comment:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create telemetry comment
+        uses: actions-ecosystem/action-create-comment@v1
+        with:
+          body: |
+            Hello world!
+          github_token: ${{ secrets.github_token }}
+
+  print-telemetry:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: ${{ env.CACHE_PATHS }}
```

Comment:
I thought you decided not to do the printing to PR right now?

Reply:
Decided to try it out now. If it wont work still, then ill give up